packages/contracts/contracts/libraries/rlp/Lib_RLPReader.sol

```diff
@@ -592,18 +592,20 @@ library Lib_RLPReader {
         }
 
         // Pick out the remaining bytes.
-        uint256 mask = 256 ** (32 - (_length % 32)) - 1;
-        assembly {
-            mstore(
-                dest,
-                or(
-                    and(mload(src), not(mask)),
-                    and(mload(dest), mask)
+        unchecked {
+            uint256 mask = 256 ** (32 - (_length % 32)) - 1;
+            assembly {
+                mstore(
+                    dest,
+                    or(
+                        and(mload(src), not(mask)),
+                        and(mload(dest), mask)
+                    )
                 )
-            )
-        }
+            }
 
-        return out;
+            return out;
```

Comment:
Minor nit: I think slightly cleaner to move this return statement outside of the `unchecked` block.